src/components/Calendar/Event/CreateEvent.js

```diff
@@ -69,16 +71,25 @@ export default function CreateEvent(props) {
                     props.onDrawerClose(res.data, 2);
             })
             .catch(err => {
-                setEventError({ ...eventError, message: "Error connecting to database. " + err });
-                setEventError({ ...eventError, hasError: true });
+                setEventError({...eventError, message: "Error connecting to database. " + err});
+                setEventError({...eventError, hasError: true});
             });
     }
 
     function handleCancel() {
         document.elementFromPoint(0, 0).click();
     }
 
-  
+    function handleTypeUpdate(e) {
```

Comment:
use the useCallBack hook here. 
https://beta.reactjs.org/reference/react/useCallback

Reply:
why? for handleTypeUpdate?